### PR TITLE
Make cell toolbar background transparent

### LINKED ISSUE #14086: Cell toobar has funny background when multiple cells are selected
## Description

When multiple cells are selected the cell toolbar buttons background is odd. This happens in both dark and light theme:

Screenshot from 2023-02-26 14-03-15

Screenshot from 2023-02-26 14-04-06

Compare to a single cell selected:

Screenshot from 2023-02-26 14-07-09

Screenshot from 2023-02-26 14-07-05

## Reproduce

1. Open notebook
2. Select multiple cells
4. See background of cell tollbar buttons is not the same as background of the cell editor

## Expected behavior

Background of cell buttons is the same (or similar to) the background of the cell/editor and does not change when cells are selected.

## Context

- JupyterLab version: 3.6.1 and 4.0.0a30

### PR DESCRIPTION
**Pull request by krassowski**
## References

Fixes #14086

## Code changes

`background-color: inherit` → `background-color: transparent`

## User-facing changes

| Before | After |
|--|--|
| Screenshot from 2023-04-15 12-58-15 | Screenshot from 2023-04-15 12-57-59 |

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by fcollonval** [comment]
> CI failures not related

I'll wait for Jason review before merging

**Review by JasonWeill — APPROVED** [review]

**Pull request merged**
